x-pack/test/reporting_api_integration/reporting_and_security/download_csv_dashboard.ts

```diff
@@ -400,62 +400,6 @@ export default function ({ getService }: FtrProviderContext) {
         };
         expect(body).to.eql(expectedBody);
       });
-
-      it(`Searches large amount of data, stops at Max Size Reached`, async () => {
-        await reportingAPI.initEcommerce();
```

Comment:
I removed this test since we know that is essentially is a benchmark test for Elasticsearch, which isn't the purpose we want for our testing.